internal/pkg/errorconcept/coredata.go

```diff
@@ -0,0 +1,59 @@
+/*******************************************************************************
+ * Copyright 2019 Dell Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *******************************************************************************/
+
+package errorconcept
+
+import (
+	"net/http"
+
+	"github.com/edgexfoundry/edgex-go/internal/core/data/errors"
+)
+
+var CoreData coreDataErrorConcept
+
+// coreDataErrorConcept contains the superfluous, duplicated errors that must be handled for CoreData
+type coreDataErrorConcept struct {
+	DBNotFound   coreDataDBNotFound
+	JsonDecoding coreDataJSONDecoding
+}
+
+type coreDataDBNotFound struct{}
```

Comment:
Would it make more sense for DBNotfound to reside in db.go and for JsonDecoding to reside in common.go?
I am thinking that this file can lead one to believe that error concepts are split up by service when they are split out by resource.

What are your thoughts?